plagiabot.py

```diff
@@ -571,7 +571,10 @@ def page_filter(self, page):
         if (rcinfo['namespace'] not in [0, 118]) and page.title() not in wikiEd_pages: return False  # only articles+drafts
         if 'length' in rcinfo:
             new_size = rcinfo['length']['new']
-            old_size = rcinfo['length']['old'] or 0
+            if 'old' in rcinfo['length']:
+                old_size = rcinfo['length']['old']
+            else:
+                old_size = 0
```

Comment:
An easier way to do this would be: `old_size = rcinfo['length'].get('old', 0)`

Reply:
I knew there must have been! Thank you.